Formula/gtk4.rb

```diff
@@ -31,19 +31,26 @@ class Gtk4 < Formula
   depends_on "libepoxy"
   depends_on "pango"
 
+  on_linux do
+    depends_on "libxkbcommon"
+  end
+
   uses_from_macos "libxslt" => :build # for xsltproc
 
   def install
     args = std_meson_args + %w[
-      -Dx11-backend=false
-      -Dmacos-backend=true
       -Dgtk_doc=false
       -Dman-pages=true
       -Dintrospection=enabled
       -Dbuild-examples=false
       -Dbuild-tests=false
     ]
 
+    on_macos do
+      args << "-Dquartz_backend=true"
```

Comment:
Is it `quartz_backend` and not `macos-backend`?

Reply:
oops, thanks

Reply:
I was looking at https://github.com/Homebrew/homebrew-core/pull/75927 and must have copied from there